### DISCUSSION TITLE
"Automatische generieren" beim erstmaligen EInsatz

### DISCUSSION THREAD

**Discussion by mcuiobroker** [Zefau/ioBroker.jarvis#1082] [Q&A]
Welche Adapter werden beim erstmaligen Einsatz eingelesen?
Also soweit ich das erkenne sind es nur 5 Adapter, die eingelesen werden. Sollte es dort nicht direkt eine Auswahl geben?

**Comment by Zefau** [comment]
Keine Auswahl. Soll so einfach wie möglich sein für den Einstieg. Den Rest kann man ja selbst importieren über den Geräte Import

**Comment by mcuiobroker** [comment]
Welche Adapter werden am Anfang denn eingelesen? Gibt es eine Liste?

**Reply by Zefau** [reply]
```
[
	'deconz',
	'hm-rpc',
	'hmip',
	'hue',
	'hue-extended',
	'shelly',
	'zigbee',
	'zwave2'
]
```
